mixins/cacheModel.js

```diff
@@ -11,7 +11,6 @@
 var debug = require('debug')('loopback:cache:CacheModel');
 
 var lib = require('../');
-var backends = lib.backends;
```

Comment:
My fault. I put it to a wrong place. Fixing it so you don't need to change this. Can you re-fork once I pushed?